### PR TITLE
Simplify and make reproducible the website setup

### PR DESCRIPTION
**Pull request by ldemailly**
We still need to make an image so it doesn’t take forever but at least
this is reproducible and working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,51 +13,26 @@ familiar with these before editing.
...
-```bash
-docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it  jekyll/jekyll sh -c "bundle install && rake test"
-```
+The `rake test` part is to make sure you are not introducing html errors or bad links, you should see
 ```
 HTML-Proofer finished successfully.
 ```
+in the output
 
-#### Side note for those on non-linux machines
```
Comment (ldemailly) [inline README.md]: I don't think we need to tell people how to install docker and the minikube part is just noise for this docs repo

File: README.md [hunk 2]
```diff
@@ -13,51 +13,26 @@ familiar with these before editing.
 To run the site locally with Docker, use the following command:
 
 ```bash
-docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll sh -c "bundle install && jekyll serve"
+docker run --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll:3.5.2 sh -c "bundle install && rake test && jekyll serve"
```
Comment (rshriram) [inline README.md]: Why did you remove the --rm ?

File: _config.yml [hunk 3]
```diff
@@ -61,3 +61,5 @@ exclude:
   - repos/*.html
   - repos/*.md
 
+repository:
+  istio/istio.github.io
```
Comment (rshriram) [inline _config.yml]: What's this for?

Comment (ldemailly) [reply _config.yml]: this is to avoid some NGO liquid error @kyessenov was getting because his git origin wasn't set

Comment (kyessenov) [reply _config.yml]: breaks on my machine without it. although i don't know if it's ok to check-in.

**Comment by ldemailly** [comment]
@rshriram see (and test !) the new version - it works well for me, subsequent start are much faster as they don't reinstall the gems only check them

**Comment by ldemailly** [comment]
@andraxylia you had issues with jekyll can you please try these updated instructions - it should work out of the box (slow for first time and fast for 2nd onward)
